item-impl/src/main/java/com/example/auction/item/impl/PItemEntity.java

```diff
@@ -51,7 +52,7 @@ private Behavior created(PItemState state) {
 
         builder.setCommandHandler(StartAuction.class, (start, ctx) -> {
             if (start.getUserId().equals(state().getItem().get().getCreator())) {
-                return ctx.thenPersist(new AuctionStarted(entityUuid(), Instant.now()), evt -> ctx.reply(Done.getInstance()));
+                return ctx.thenPersist(new AuctionStarted(entityUuid(), Instant.now()), done(ctx));
```

Comment:
Think it might be good to implement `ctx.done()` in the framework as a convenience.